packages/Webkul/Shop/src/Http/Controllers/ProductsCategoriesProxyController.php

```diff
@@ -37,10 +37,12 @@ public function index(Request $request)
 
         $slugOrPath = urldecode($slugOrPath);
 
-        // support url for chinese, japanese, arbic and english with numbers.
+        // support url for chinese, japanese, arabic and english with numbers.
         if (preg_match('/^([\x{0621}-\x{064A}\x{4e00}-\x{9fa5}\x{3402}-\x{FA6D}\x{3041}-\x{30A0}\x{30A0}-\x{31FF}_a-z0-9-]+\/?)+$/u', $slugOrPath)) {
             if ($category = $this->categoryRepository->findByPath($slugOrPath)) {
```

Comment:
I think the child category can be made in such a way that it can be eager-loaded. Rather than again sending the id to hit one more query.

Reply:
Will handle it later. I am approving this.